.github/workflows/ci.yaml

```
@@ -71,60 +70,18 @@ jobs:
+        run: echo "No Dockerfile changes, skipping build."
 
-      - name: Build and push Docker image
+      - name: Docker build (CI only, no push)
         if: steps.changed-files.outputs.dockerfile_any_changed == 'true'
-        uses: docker/build-push-action@v6
+        uses: ./.github/actions/docker-build
         with:
-          context: .
-          file: apps/api/Dockerfile
+          version: dev
+          env: dev
```

actionlint (1.7.4)

input "env" is not defined in action "Docker Build & Push" defined at "./.github/actions/docker-build". available inputs are "github_token", "push", "tag_latest", "turbo_api", "turbo_team", "turbo_token", "version"

(action)